src/Icons/src/Iconify.php

```diff
@@ -26,7 +26,8 @@
 final class Iconify
 {
     public const API_ENDPOINT = 'https://api.iconify.design';
-
+    public const XMLNS_URL = 'http://www.w3.org/2000/svg';
```

Comment:
```suggestion
    public const XMLNS_URL = 'https://www.w3.org/2000/svg';
```

Reply:
Let's make it private too, there is no use-case to make it public (and event for tests) 

Also, we can prefix it by `ATTR_` as well